src/lavinmq/http/controller/queues.cr

```diff
@@ -41,6 +42,25 @@ module LavinMQ
           end
         end
 
+        get "/api/queues/:vhost/:name/unacked" do |context, params|
+          with_vhost(context, params) do |vhost|
+            refuse_unless_management(context, user(context), vhost)
+            unacked_messages = Array(UnackedMessage).new
+            @amqp_server.vhosts[vhost].queues[params["name"]].@consumers.each do |c|
+              c.unacked_messages.select { |u| u.queue.name == params["name"] }.each do |u|
+                if consumer = u.consumer
```

Comment:
what about messages that are unacked via BasicGet, we don't want display them?

Reply:
Hmm, yeah we probably do! Not sure how we would do that though, we only really save them as increasing/decreasing `unacked_count` and `unacked_bytesize` now right? So we don't really have enough information. 
Maybe we could add an Array(UnackedMessage) to queues where we save any unacked via BasicGet? 

Something like this maybe? (but actually working 😄 )
```
queue_unacked = Array(UnackedMessage.new

private def mark_unacked(sp, &)
      @log.debug { "Counting as unacked: #{sp}" }
      @unacked_lock.synchronize do
        @unacked_count += 1
        @unacked_bytesize += sp.bytesize
        queue_unacked.push UnackedMessage.new(..) # track unacked messages:
```

and do something like `queue_unacked.delete(msg)` in `basic_ack `& `basic_reject` and in the rescue in `mark_unacked`

WDYT?

Reply:
yes, let's store basicget messages in a `Deque` in `Queue`, shouldn't be that many

Reply:
no need to store the unacked messages deliviered to consumers.

Reply:
I have something that kinda works in my branch now, but not really sure what we want to store/display for the basic_get unacked messages. They don't really have a consumer_tag, channel_name or delivery_tag. Guess we can have the time for how long it's been unacked at least.